### DISCUSSION TITLE
How to change keybinds?

### DISCUSSION THREAD

**Discussion by uieqwhak** [shedaniel/RoughlyEnoughItems#1526] [Q & A] [answered]
"Alternative to Just Enough Items/JEI." yet there's no way to rebind keys? I like this mod, but this lacking feature alone is the only reason I switch to JEI. I checked every setting, and the whole controls menu, is there actually a way to change keybinds? I would happily switch back to this amazing mod if I could change keybinds.

**Comment by rozbrajaczpoziomow** [comment, marked as answer]
IIRC, REIs own settings menu has rebindable keybinds

**Reply by PinguinKeks** [reply]
it doesn't, at least not on fabric

**Reply by DeceptionXD** [reply]
Solution .download the newest version of JEI (15.20.0.116), and it gives you JEI keybinds
